bridge_adaptivity/bridge_lti/tests/test_exeptions.py

```diff
@@ -16,44 +19,38 @@
 
 
 class RaisedExceptionUsesCustomTemplateTest(BridgeTestCase):
+    """
+    Test incorrect working LTI provider.
+    """
+
     def setUp(self):
         super().setUp()
         self.rf = RequestFactory()
-        self.correct_kw = {'collection_slug': self.collection1.slug, 'group_slug': self.test_cg.slug}
-        self.not_correct_kw = {'collection_slug': self.collection2.slug, 'group_slug': self.test_cg.slug}
+        self.correct_kw = {'collection_order_slug': self.collection_order1.slug}
+        self.not_correct_kw = {'collection_order_slug': self.collection_order1.slug + "_wrong"}
         self.url = reverse('lti:launch', kwargs=self.correct_kw)
         self.not_correct_url = reverse('lti:launch', kwargs=self.not_correct_kw)
 
     @override_settings(DEBUG=False)
-    def test_learner_flow_with_incorrect_collection_slug(self):
-        """Check if learner_flow function is called with incorrect collection_slug raise proper exception."""
+    def test_learner_flow_with_incorrect_collection_order_slug(self):
+        """
+        Check if learner_flow function is called with incorrect collection_slug raise proper exception.
+        """
         request = self.rf.post(self.url)
         with pytest.raises(Http404):
             learner_flow(
                 request,
                 lti_consumer=None,
                 tool_provider=None,
-                collection_slug=1000,
-                group_slug=self.test_cg.slug,
-            )
-
-    @override_settings(DEBUG=False)
-    def test_learner_flow_with_collection_not_in_passed_group(self):
-        """Check if learner_flow function is called with incorrect collection_slug raise proper exception."""
-        request = self.rf.post(self.not_correct_url)
-        with pytest.raises(Http404):
-            learner_flow(
-                request,
-                lti_consumer=None,
-                tool_provider=None,
-                collection_slug=self.not_correct_kw['collection_slug'],
-                group_slug=self.not_correct_kw['group_slug'],
+                collection_order_slug=self.not_correct_kw["collection_order_slug"]
             )
 
     @mock.patch('lti.contrib.django.DjangoToolProvider.from_django_request')
     @override_settings(DEBUG=False)
     def test_client_post_with_incorrect_collection_slug_test(self, from_django_request):
```

Comment:
Please ensure test's name is correctly map test case.

Reply:
I changed the name of this test (test_client_post_with_incorrect_collection_order_slug_test) and used self.not_correct_url insted of self.url. 
This test checks case if collection_order_slug is wrong. Mocks are used for returning 'false' value in this line: https://github.com/harvard-vpal/bridge-adaptivity/pull/225/files#diff-2955dc1a69071d45a3aaf904bae1b12bR37